sim/src/main/scala/spinal/sim/XSimBackend.scala

```diff
@@ -208,9 +230,9 @@ class XSimBackend(config: XSimBackendConfig) extends Backend {
 
     val vivadoScriptPath = scriptPath.replace("\\", "/")
     val command = {
-      val baseCommand = s"vivado -mode batch -source $vivadoScriptPath"
+      val baseCommand = s""" vivado -mode batch -source "$vivadoScriptPath" """.trim
       if (isWindows) {
-        s"sh -c \'${baseCommand}\'"
+        s"cmd /c $baseCommand"
```

Comment:
so do you use cmd console by default?

Reply:
yes, as mentioned in the issue a few times : 
my goal was using native cmd console by default to call everything that can be : 
`vivado.bat`, and its generated `compile.bat` and `elaborate.bat` : 
so whatever if user starts sbt from cmd or from msys shell :
- all `.bat` will be run by cmd
- the `g++` command to build `spinal_xsim.dll`